database/schema/V1.0__schema.sql

```diff
@@ -1,126 +1,1356 @@
-/*
-    Schema V1.0: Initial schema
-*/
-
-
-/* 
-    create schemas
-*/
-create schema reference;
-create schema configuration;
-
-/*
-    create tables - reference
-*/
-
-create table reference.organisation
-(
-    organisation_id serial not null,
-    organisation_name varchar(100) not null,
-    ods_code varchar(10) not null,
-    org_status varchar(50) not null,
-    org_record_class varchar(10) not null,
-    postcode varchar(10) not null,
-    last_change_date timestamp not null,
-    primary_role_id varchar(10) not null,
-    primary_role_description varchar(100) not null,
-    organisation_link varchar(1000) not null,
-    added_date timestamp not null,
-    last_sync_date timestamp not null,
-
-    constraint reference_organisation_organisationid_pk primary key (organisation_id),
-    constraint reference_organisation_odscode_uq unique (ods_code), 
-    constraint reference_organisation_odscode_ck_1 check (char_length(trim(ods_code)) > 0),
-    constraint reference_organisation_odscode_ck_2 check (upper(ods_code) = ods_code),
-    constraint reference_organisation_orgstatus_ck check (char_length(trim(org_status)) > 0),
-    constraint reference_organisation_orgrecordclass_ck check (char_length(trim(org_record_class)) > 0),
-    constraint reference_organisation_postcode_ck check (char_length(trim(postcode)) > 0),
-    constraint reference_organisation_primaryroleid_ck check (char_length(trim(primary_role_id)) > 0),
-    constraint reference_organisation_primary_roledescription_ck check (char_length(trim(primary_role_description)) > 0),
-    constraint reference_organisation_organisationlink_ck check (char_length(trim(organisation_link)) > 0),
-    constraint reference_organisation_organisationname_ck check (char_length(trim(organisation_name)) > 0)
+--
```

Comment:
Can we remove system generated comments.

Reply:
Done